src/main/java/com/exactpro/th2/act/ActHandler.java

```diff
@@ -441,6 +484,27 @@ private EventID createAndStoreParentEvent(PlaceMessageRequestOrBuilder request,
         }
     }
 
+    private EventID createAndStoreParentEvent(MultiSendRequest request, String actName, Status status) throws IOException {
+        long startTime = System.currentTimeMillis();
+
+        Event event = start()
+                .name(actName + ' ' + request.getMessages(0).getMetadata().getId().getConnectionId().getSessionAlias())
```

Comment:
I think we should check the size of the message list before accessing the first element

Reply:
I've added the check to the beginning of `multiSendMessage` method.

Reply:
Sorry, missed that